formulas/glance/install.sls

```diff
@@ -41,3 +41,8 @@ glance_packages:
       - python3-openstackclient
 
 {% endif %}
+
+pip_packages:
```

Comment:
Can you add the 'env' key here?  Some OS are going to support python2 for awhile, so we want to be explicit that we want to use python3.  There are a few examples in the other 'install.sls' files for stuff that uses pip.